<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataReader</class>
 <widget class="QMainWindow" name="DataReader">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataReader</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="reppath">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browse_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>43</height>
             </size>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="open_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>Open report</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Reading the report file:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="progress">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>0%</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCustomPlot" name="customPlot" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>340</height>
           </size>
          </property>
          <property name="title">
           <string>Parameters:</string>
          </property>
          <widget class="QLabel" name="label_12">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>210</y>
             <width>81</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>TOTAL_TIME:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>121</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>CUDA_Y_THREADS:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_26">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>130</y>
             <width>51</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>Y_MAX:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_14">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>230</y>
             <width>101</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>STEP_LENGTH:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_18">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>170</y>
             <width>31</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>N_Y:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>190</y>
             <width>31</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>TAU:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_24">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>110</y>
             <width>51</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>X_MAX:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_19">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>150</y>
             <width>31</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>N_X:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>91</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>MPI_NODES_X:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>50</y>
             <width>91</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>MPI_NODES_Y:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_15">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>70</y>
             <width>121</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>CUDA_X_THREADS:</string>
           </property>
          </widget>
          <widget class="QLabel" name="total_time_2">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>210</y>
             <width>81</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="tau_2">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>190</y>
             <width>31</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="n_x_2">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>150</y>
             <width>31</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="cuda_x_threads_2">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>70</y>
             <width>121</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="n_y_2">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>170</y>
             <width>31</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="step_length_2">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>230</y>
             <width>101</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="cuda_y_threads_2">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>90</y>
             <width>121</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="mpi_nodes_y_2">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>50</y>
             <width>91</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="mpi_nodes_x_2">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>30</y>
             <width>91</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="y_max_2">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>130</y>
             <width>51</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="x_max_2">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>110</y>
             <width>51</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>290</y>
             <width>151</width>
             <height>41</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_21">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>270</y>
             <width>111</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>Property to draw:</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSlider" name="timeSlider">
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="2">
         <widget class="QPushButton" name="anthr_report_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Open another report</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="time_lbl">
          <property name="text">
           <string>1 / 1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../../utilities/QCustomPlot/include/Qcustomplot.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
